### DISCUSSION TITLE
Run-time combination of SUMO and ns-3

### DISCUSSION THREAD

**Discussion by pzambare3** [eclipse/mosaic#372] [Q&A]
I want combine SUMO and NS3.Can you please provide proper step-by-step guide to integrate sumo with ns3?

**Comment by schwepmo** [comment]
Unfortunately, we don't have the capacity to provide step-by-step guidance. However, I can give you some hints where to start reading:

- https://eclipse.dev/mosaic/docs/simulators/network_simulator_ns3/ Link to the ns3 documentation on our page, which provides installation instructions and basic configurations
- The [Tiergarten Scenario](https://github.com/eclipse/mosaic/tree/main/bundle/src/assembly/resources/scenarios/Tiergarten) which can be changed to use ns3 by adjusting the `scenario_config.json` to use the ns3 federate

**Comment by pzambare3** [comment]
@schwepmo: Thanks!
